### PR TITLE
Child() returns children if no arguments are provided

### PR DESCRIPTION
**Pull request by darkcoderrises**
Added Functionality to child(). Have not written the tests yet.

### CODE REVIEW

File: lib/addons/p5.dom.js [hunk 1]
```diff
@@ -1031,6 +1031,9 @@
    * </code></div>
    */
   p5.Element.prototype.child = function(c) {
+    if (c == null){
+      return this.eld.childNodes
```
Comment (lmccart) [inline lib/addons/p5.dom.js]: elt?

Comment (darkcoderrises) [inline lib/addons/p5.dom.js]: oh, yeah. Will fix this.

Comment (darkcoderrises) [inline lib/addons/p5.dom.js]: But does this returns the desired elements?

File: lib/addons/p5.dom.js [hunk 2]
```diff
@@ -1031,6 +1031,9 @@
    * </code></div>
    */
   p5.Element.prototype.child = function(c) {
+    if (c == null){
```
Comment (lmccart) [inline lib/addons/p5.dom.js]: this should be `===`

**Comment by darkcoderrises** [comment]
Where should I write the test?

**Comment by lmccart** [comment]
looks great, thanks! if you'd like to add a test, you can add it to this file: https://github.com/processing/p5.js/blob/master/test/unit/addons/p5.dom.js

**Pull request merged**